### PR TITLE
[MIG] partner_financial_risk > account_financial_risk: Migration to 11.0

### PR DESCRIPTION
**Pull request by carlosdauden**
@Tecnativa

### CODE REVIEW

**Review by chienandalu — COMMENTED** [review]
Runbot is failing (502 Bad Gateway)

**Review by pedrobaeza — APPROVED** [review]
I'm not sure why runbot is not building a working instance, but I have tried it locally.

File: account_financial_risk/__init__.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+from . import models
+from . import wizard
```
Comment (sergio-teruel) [inline account_financial_risk/__init__.py]: Use plural form 
[https://github.com/OCA/maintainer-tools/tree/master/template/module](url)

File: account_financial_risk/data/account_financial_risk_data.xml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+<record id="ir_cron_due_invoice_every_day" model="ir.cron">
+    <field name="name">Financial risk: Process due invoices</field>
+    <field name="model_id" ref="model_res_partner"/>
+    <field name="state">code</field>
+    <field name="code">model.process_unpaid_invoices()</field>
+    <field name="user_id" ref="base.user_root" />
+    <field name="interval_number">1</field>
+    <field name="interval_type">days</field>
+    <field name="numbercall">-1</field>
+    <!--<field name="doall" eval="False"/>-->
```
Comment (sergio-teruel) [inline account_financial_risk/data/account_financial_risk_data.xml]: Remove if not used

File: account_financial_risk/views/res_config_view.xml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+<!-- Copyright 2016-2018 Tecnativa - Carlos Dauden
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+
+<record id="res_config_settings_view_form" model="ir.ui.view">
+    <field name="name">res.config.settings.view.form.inherit.partner.financial.risk</field>
+    <field name="model">res.config.settings</field>
+    <field name="inherit_id" ref="account.res_config_settings_view_form"/>
+    <field name="arch" type="xml">
+        <xpath expr=".//div[@id='account_followup']/.." position="inside">
```
Comment (sergio-teruel) [inline account_financial_risk/views/res_config_view.xml]: Remove dot.

File: account_financial_risk/views/res_config_view.xml [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+            <div class="col-xs-12 col-md-6 o_setting_box" title="These taxes are set in any new product created.">
+                <div class="o_setting_left_pane"/>
+                <div class="o_setting_right_pane">
+                    <label string='Financial Risk'/>
+                    <span class="fa fa-lg fa-building-o" title="Values set here are company-specific." groups="base.group_multi_company"/>
+                    <div class="text-muted">
+                        Set options applied to financial risk
+                    </div>
+                    <div class="content-group">
+                        <div class="row mt16" title="">
```
Comment (sergio-teruel) [inline account_financial_risk/views/res_config_view.xml]: Empty title attribute?

File: account_financial_risk/views/res_partner_view.xml [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+<odoo>
+
+    <record id="res_partner_view_risk" model="ir.ui.view">
+        <field name="name">res.partner.view.risk</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form" />
+        <field name="priority" eval="20"/>
+        <field name="groups_id" eval="[(4, ref('account.group_account_manager'))]"/>
+        <field name="arch" type="xml">
+
```
Comment (sergio-teruel) [inline account_financial_risk/views/res_partner_view.xml]: Remove blank line

File: account_financial_risk/views/res_partner_view.xml [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+            <page name="sales_purchases" position="after">
+                <page name="financial_risk" string="Financial Risk" attrs="{'invisible': [('customer','=',False)]}">
```
Comment (sergio-teruel) [inline account_financial_risk/views/res_partner_view.xml]: Add white spaces in the domain

**Review by sergio-teruel — CHANGES_REQUESTED** [review]

**Review by oihane — APPROVED** [review]
Technically reviewed

**Comment by pedrobaeza** [comment]
@carlosdauden can you attend the remaining comments?

**Comment by carlosdauden** [comment]
@sergio-teruel Changes attended

**Comment by rafaelbn** [comment]
@carlosdauden travis is :red_circle: just missing travis! :+1:

**Comment by pedrobaeza** [comment]
@moylop260 do you have any clue why Travis is failing in this 11.0 branch due to absence of PyPDF2 library? https://travis-ci.com/OCA/credit-control/jobs/130482431#L600

**Comment by pedrobaeza** [comment]
@moylop260 I see that the .travis.yml is not correct. Fixing it.

**Comment by pedrobaeza** [comment]
Finally green! Merging

**Pull request merged**